pkgs/development/python-modules/pleroma-bot/default.nix

```
@@ -0,0 +1,37 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, pytest
+, requests-mock
+, oauthlib
+, requests_oauthlib
+, requests
+, pyaml
+}:
+
+buildPythonPackage rec {
+  pname = "pleroma-bot";
+  version = "0.8.6";
+
+  src = fetchFromGitHub {
+    owner  = "robertoszek";
+    repo   = "pleroma-bot";
+    rev    = "0.8.6";
+    sha256 = "1q0xhgqq41zbqiawpd4kbdx41zhwxxp5ipn1c2rc8d7pjyb5p75w";
+  };
+
+  checkInputs = [ pytest ];
+  checkPhase = "pytest";
```

Looks like that `pytestCheckHook` could be used.

```suggestion
  checkInputs = [ pytestCheckHook ];
```

---

Yep, I just tried and executes all tests as expected. Thanks for the suggestion